bakery_cli/scripts/genmetadata.py

```diff
@@ -424,7 +424,8 @@ def check_monospace(familydir):
     # by casting list to python sets.
     return len(set(glyphwidths)) == 1
 
-
+# FS: This function is now unused
+# It was only used to set the now-deprecated size field in font metadata
```

Comment:
@davelab6  I want to delete this deprecated function (getSize) that is only used by metadata.size